jwst/lib/set_telescope_pointing.py

```diff
@@ -1422,10 +1422,18 @@ def populate_model_from_siaf(model, siaf):
         model.meta.wcsinfo.wcsaxes = 2
         model.meta.wcsinfo.cunit1 = "deg"
         model.meta.wcsinfo.cunit2 = "deg"
-        model.meta.wcsinfo.crpix1 = siaf.crpix1
-        model.meta.wcsinfo.crpix2 = siaf.crpix2
-        model.meta.wcsinfo.cdelt1 = siaf.cdelt1 / 3600  # in deg
-        model.meta.wcsinfo.cdelt2 = siaf.cdelt2 / 3600  # in deg
+        model.meta.wcsinfo.crpix1 = 0.0  # initialize to default
+        model.meta.wcsinfo.crpix2 = 0.0  # initialize to default
+        model.meta.wcsinfo.cdelt1 = 1.0  # initialize to default
+        model.meta.wcsinfo.cdelt2 = 1.0  # initialize to default
+        if siaf.crpix1:
+            model.meta.wcsinfo.crpix1 = siaf.crpix1
+        if siaf.crpix2:
+            model.meta.wcsinfo.crpix2 = siaf.crpix2
+        if siaf.cdelt1:
+            model.meta.wcsinfo.cdelt1 = siaf.cdelt1 / 3600  # in deg
+        if siaf.cdelt2:
+            model.meta.wcsinfo.cdelt2 = siaf.cdelt2 / 3600  # in deg
```

Comment:
The SIAF tuple has default values of `None` for all members, used in case the siaf database does not have values. This allows to writ ehe above lines as
```
model.meta.wcsinfo.crpix1 = siaf.crpix1 or 0
model.meta.wcsinfo.crpix2 = siaf.crpix2 or 0
model.meta.wcsinfo.cdelt1 = siaf.cdelt1 / 3600  or 1# in deg
model.meta.wcsinfo.cdelt2 = siaf.cdelt2 / 3600 or 1 # in deg
```

Reply:
The above syntax involving crpix1 and crpix2 works fine when they're None, but it doesn't work for the cdelts, because it first tries to evaluate ``siaf.cdelt1 / 3600`` and throws the same ``TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'``